2.7/s2i/bin/assemble

```diff
@@ -14,6 +14,9 @@ echo "---> Copying application source ..."
 cp -Rf /tmp/src/. ./
 
 if [[ -f requirements.txt ]]; then
+  if [ -n "$PIP_INDEX_URL" ]; then
+    export PIP_INDEX_URL=${PIP_INDEX_URL}
```

Comment:
Hey Ben. My bad on this. Originally, I tested to inject PIP_INDEX_URL directly via env var and it didn't work for some reasons. I thought it was a out-of-the-scope issue so I added those codes this PR. Then, I retested and it was working fine so I removed the code. This PR should be docs change to reflect the env var availability. I will fix this ASAP.